Learning_Guide/Kickoff.md

```diff
@@ -6,106 +6,41 @@ The purpose of a project is for the team to achieve the goal through practicing
+3. It is a means for you to practice **project estimation**, **spec'ing**, and **goal setting**. These are critical skill for any software developer to develop (and really difficult ones).
 4. It challenges you and your team to stay in your Zone of Proximal Development (ZPD). Without a good challenge, your learning will not be optimized.
 
-#### Preparing for the contract
-
-Before writing your Learning Contract, it's good practice to checkin with your team about the following:
-
-- How familiar are they with this goal? Have they done it before? What have they learned?
-- How available are they for the week? Do they have any planned personal time off?
-- How familiar are they with the technology stack suggested by the goal (if any)?
-- Is this is a normal week, or are there any holidays / special events that will shorten it?
-
-Once you've surveyed your teams' skill and availability, it's time to solidify the specs:
-
-**You should aim to write specs that are clear, and detailed, and challenging. They should put the entire team in their ZPD for the rest of the week. The goal is to maximize quantity and quality of software shipped. The Learning Contract should challenge everyone on the team, *not* to make sure everyone is comfortable.**
-
-To create a Learning Contract, create a new `contract` branch:
-
-```
-git branch contract
-git checkout contract
-git push origin contract
-```
-
-Copy the text of the goal into the `README.md` file.
-
-
-#### Specifications and Quality Criteria
-
-After creating the Learning Contract, the team must edit the document to define the scope of features and quality criteria for their project.
-
-The scope of features is determined by the set of specifications for the project. It is used to measure _completeness_.
-
-The quality criteria are rules that govern how the artifact should be produced. They are used to measure _quality_.
-
-To define the scope, add, edit, or remove specifications from the list. The team is accountable for producing an artifact that satisfies _all_ of the specifications, so be realistic.
-
-Discuss the specs with your team, and adjust them as you see fit to get to a scope that will provide an appropriate level of difficulty and "stretch".
-
-To set quality criteria, choose 2 or 3 rules or guidelines that, as a team, you agree to follow to ensure a quality artifact. Add these to the Learning Contract.
-
-For example, a quality criteria might be "Follow object-oriented principles in all program design" or "Write good commit messages using these rules: http://chris.beams.io/posts/git-commit/".
+Review the Learning Contract included in the goal. You may need to improve it if the specs are unclear or the quality rubric is incomplete. If so, make changes in a pull request and then submit your PR to [prrr](http://prrr.apps.learnersguild.org/).
```

Comment:
Hmmm... a little tricky here. Maybe make it clear somehow that this is an exception. 
Also, they should submit the changes to the goal librarian as well